packages/itwinui-css/src/menu/menu.scss

```diff
@@ -193,3 +206,18 @@ $iui-active-outline: thin solid var(--iui-color-border-accent);
     margin-left: var(--iui-size-xs);
   }
 }
+
+@mixin iui-multilevel {
+  @include iui-list-menu;
```

Comment:
This `@include` causes potentially unnecessary duplication of CSS. If you only need a few declarations from the original mixin, it might be better to move those out into a separate intermediate mixin that can be used in both `iui-multilevel` and `iui-list-menu`.

The same comment also applies below to `@include iui-menu-item`.

Reply:
Updated. I tried to make it understandable.